src/translations/en.json

```diff
@@ -8126,6 +8127,11 @@
                 "info_text_2": "If the historic statistic values have a wrong unit, you can update the units of the old values. The values will not be updated.",
                 "info_text_3": "Otherwise you can choose to delete all historic statistic values, and start over."
               },
+              "mean_type_changed": {
+                "title": "The mean type has changed",
+                "info_text_1": "The mean type of ''{name}'' ({statistic_id}) changed from ''{metadata_mean_type}'' to ''{state_mean_type}''.",
```

Comment:
The LTS are suppressed until the issue is fixed. Users can either delete the old stats data or change the mean type back.
Should we mention this here somewhere?

Reply:
Yeah, will update the text 👍